pkg/option/config.go

```diff
@@ -3360,9 +3360,8 @@ func (c *DaemonConfig) validateVTEP() error {
 		len(vtepEndpoints) != len(vtepMACs) {
 		return fmt.Errorf("VTEP configuration must have the same number of Endpoint, VTEP and MAC configurations (Found %d endpoints, %d MACs, %d CIDR ranges)", len(vtepEndpoints), len(vtepMACs), len(vtepCIDRs))
 	}
-	//Todo: resolve github issue 18616 to lift the maximum 2 VTEP limit
-	if len(vtepEndpoints) > 2 {
-		return fmt.Errorf("VTEP must not exceed 2 VTEP devices (Found %d VTEPs)", len(vtepEndpoints))
+	if len(vtepEndpoints) > 8 {
```

Comment:
It might be nice to define this `8` in `pkg/defaults` instead, so then the same definition can be used both here in the `pkg/option` as well as `pkg/maps/vtep`.

Reply:
done